.github/workflows/python-package.yml

```diff
@@ -17,11 +17,16 @@ on:
   merge_group:
     types: [checks_requested]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.head_ref }}
+  cancel-in-progress: ${{ ! (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/release/')) }}
+
 jobs:
   build:
 
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
What's the reason for this line?

Reply:
it was quite confusing to see jobs canceled anot be able to know if all is fine except one